templates/react-native/src/id.ts.twig

```diff
@@ -0,0 +1,9 @@
+export class ID {
+    public static custom(id: string): string {
+        return id
+    }
+    
+    public static unique(): string {
+        return 'unique()'
+    }
```

Comment:
This has been updated to generate the ID client-side:

https://github.com/appwrite/sdk-generator/blob/ad80d80e18f0cda981e1bddbf0841a805632caa0/templates/web/src/id.ts.twig#L1-L28

Reply:
updated